SeaPublicWebsite/Helpers/PropertyDataHelper.cs

```diff
@@ -5,115 +5,6 @@ namespace SeaPublicWebsite.Helpers
 {
     public class PropertyDataHelper
     {
-        // User has a solid floor
-        // OR user do not know and the EPC states they have a solid floor
-        // OR user does not know and property built after 1950
-        // User year built is used if provided, over the EPC construction date
-        public static bool HasSolidFloor(PropertyData propertyData)
-        {
-            return propertyData.FloorConstruction == FloorConstruction.SolidConcrete
-                   || propertyData.FloorConstruction == FloorConstruction.DoNotKnow
-                   && (propertyData.Epc?.FloorConstruction == FloorConstruction.SolidConcrete
-                       || propertyData.Epc?.FloorConstruction == null
-                       && (!propertyData.YearBuilt.HasValue &&
-                           propertyData.Epc?.ConstructionAgeBand >= HomeAge.From1950To1966 || propertyData.YearBuilt >= 1950));
-        }
```

Comment:
Only used for hints when the user had no EPC. Since we are not doing that anymore, these can be safely deleted.